validate.php

```diff
@@ -103,9 +108,11 @@
     print_fail('You need to run this script as root' . (isset($config['user']) ? ' or '.$config['user'] : ''));
 }
 
-if ($config['update_channel'] == 'master' && $cur_sha != $versions['github']['sha']) {
-    $commit_date = new DateTime('@'.$versions['local_date'], new DateTimeZone(date_default_timezone_get()));
-    print_warn("Your install is out of date, last update: " . $commit_date->format('r'));
+if ($git_found === true) {
```

Comment:
Alright, as long as we are doing it purposefully, you are the master of the php code ;)